readme.md

```diff
@@ -2,3 +2,59 @@ Ansible Atlantis Role
 ========================
 
 Configure Atlantis role.
+
+## Role Variables
+
+- `atlantis_server_config_file`: The name of the atlantis server side config file (Optional)
+
+## Requirements
+
+- Docker and Docker Compose
+- A `atlantis_config.yml` file in your files folder. This file contains all server side config for Atlantis
+- A `atlantis_env.j2` template in your templates folder. This file contains all environment variables for Atlantis and Terraform.
+Atlantis requires `ATLANTIS_ATLANTIS_URL` and `ATLANTIS_REPO_ALLOWLIST` to be set. You also need to specify the all required env variables for your VCS.
+This file contains also all secreted needed by Terraform. 
+
+Example: 
+```bash
+ATLANTIS_ATLANTIS_URL=
+ATLANTIS_REPO_ALLOWLIST=
+
+ATLANTIS_GH_USER=
+ATLANTIS_GH_TOKEN=
+ATLANTIS_GH_WEBHOOK_SECRET=
+
+DIGITALOCEAN_TOKEN=
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+```
```

Comment:
Sorry, the suggestion is not working as expected, because of the code part. That's why I will post it like this:

```bash
ATLANTIS_ATLANTIS_URL={{ atlantis['url'] }}
ATLANTIS_REPO_ALLOWLIST={{ atlantis['allowed_repos'] }}

ATLANTIS_GH_USER={{ atlantis['github']['user'] }}
ATLANTIS_GH_TOKEN={{ atlantis['github']['token'] }}
ATLANTIS_GH_WEBHOOK_SECRET={{ atlantis['github']['webhook_secret'] }}

DIGITALOCEAN_TOKEN={{ atlantis['digitalocean']['api_key'] }}
AWS_ACCESS_KEY_ID={{ atlantis['terraform']['aws']['access_key_id'] }}
AWS_SECRET_ACCESS_KEY={{ atlantis['terraform']['aws']['secret_access_key'] }}
```

The variables within the brackets are Ansible variables. You could store these secrets for example using Ansible Vault.